### ISSUE TITLE
Which data set are the pretrained weight files trained on?

### ISSUE THREAD

**Issue by GZhengyang** [cliffbb/OEM-Fewshot-Challenge#4]
Are the pre-training weights provided directly trained based on the 258 image-label pairs of the trainset provided by the competition party, or are they trained based on the coco data set or PASCAL-5i data set in DLAM?

**Comment by cliffbb** [comment]
@GZhengyang Yes, the baseline is pre-trained only on the 258  image-label pairs of the trainset. 
But, in addition to the 256 images, you can train your backbone on any dataset except **remote sensing** datasets.
